internal/controlplane/handlers_repositories_test.go

```diff
@@ -17,223 +17,132 @@ package controlplane
 import (
 	"context"
 	"database/sql"
-	"encoding/json"
 	"errors"
 	"net/http"
-	"reflect"
 	"testing"
 
-	"github.com/ThreeDotsLabs/watermill/message"
 	"github.com/go-git/go-git/v5"
 	"github.com/google/go-github/v56/github"
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
 	"go.uber.org/mock/gomock"
 	"golang.org/x/oauth2"
-	"google.golang.org/protobuf/proto"
 
 	mockdb "github.com/stacklok/minder/database/mock"
-	"github.com/stacklok/minder/internal/config/server"
 	mockcrypto "github.com/stacklok/minder/internal/crypto/mock"
 	"github.com/stacklok/minder/internal/db"
 	"github.com/stacklok/minder/internal/engine"
 	ghprovider "github.com/stacklok/minder/internal/providers/github"
 	"github.com/stacklok/minder/internal/providers/ratecache"
 	ghrepo "github.com/stacklok/minder/internal/repositories/github"
 	mockghrepo "github.com/stacklok/minder/internal/repositories/github/mock"
-	mockghhook "github.com/stacklok/minder/internal/repositories/github/webhooks/mock"
-	"github.com/stacklok/minder/internal/util/ptr"
 	pb "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
 	provinfv1 "github.com/stacklok/minder/pkg/providers/v1"
 )
 
 func TestServer_RegisterRepository(t *testing.T) {
 	t.Parallel()
 
-	accessToken := "AUTH"
-
-	tests := []struct {
-		name    string
-		req     *pb.RegisterRepositoryRequest
-		repo    github.Repository
-		repoErr error
-		want    *pb.RegisterRepositoryResponse
-		events  []message.Message
-		wantErr bool
-	}{{
-		name: "register repository, invalid upstream ID",
-		req: &pb.RegisterRepositoryRequest{
-			Provider: "github",
-			Repository: &pb.UpstreamRepositoryRef{
-				Owner:  "test",
-				Name:   "a-test",
-				RepoId: 31337,
```

Comment:
What happened to the test for ignoring the `RepoId` in the request in favor of looking it up from GitHub?  This is [GHSA-q6h8-4j2v-pjg4](https://github.com/stacklok/minder/security/advisories/GHSA-q6h8-4j2v-pjg4), so I'd like to continue to have a test for it somewhere.